hipamd/hip-config-amd.cmake

```diff
@@ -63,7 +63,7 @@ if(NOT HIP_CXX_COMPILER)
 endif()
 
 if(NOT WIN32)
-  find_dependency(AMDDeviceLibs)
+  find_dependency(AMDDeviceLibs HINTS ${ROCM_PATH})
```

Comment:
Shouldn't this be PATHS instead of HINTS? 
According to https://cmake.org/cmake/help/latest/command/find_package.html

Reply:
not sure about this myself, I have discussed this some time with others as well and it wasn't quite clear if it should be `PATHS ${ROCM_PATH}`, `HINTS ${ROCM_PATH}` or `PATHS ${ROCM_PATH} HINTS /opt/rocm`

I'll defer here to reviewer choice :)

Reply:
Given that (according to cmake doc): _HINTS should be paths computed by system introspection, such as a hint provided by the location of another item already found. Hard-coded guesses should be specified with the PATHS option._ 
I think the correct option here is `HINTS ${ROCM_PATH} PATHS "/opt/rocm"`

Also a note that with the proposed changes it is not necessary for user to set the ROCM_PATH environment variable. i.e. just provide the path to the rocm installation would also work i.e.

`find_package(HIP CONFIG PATHS "/opt/rocm")`

I have transferred the changes to an internal PR and it should appear in this repo shortly after it is merged. Will then close this PR.

Reply:
great, thank you!